pqarrow/arrow.go

```diff
@@ -372,6 +372,11 @@ func ParquetNodeToType(n parquet.Node) arrow.DataType {
 func parquetNodeToType(n parquet.Node) (arrow.DataType, func(b array.Builder, numValues int) valueWriter) {
 	t := n.Type()
 	lt := t.LogicalType()
+
+	if lt == nil {
+		panic("unsupported type")
```

Comment:
We'd rather not panic anywhere. We should change the function signature to return an error in this case.

Reply:
I've fixed it.
https://github.com/polarsignals/arcticdb/pull/70/commits/3eee2c5c38b625024b7100edd828f4b1cec5eecc